pkg/netpol/diff/diff.go

```diff
@@ -203,6 +221,13 @@ func (c *ConnsPair) isSrcOrDstPeerIPType(checkSrc bool) bool {
 	return (checkSrc && src.IsPeerIPType()) || (!checkSrc && dst.IsPeerIPType())
 }
 
+// helpers to check if a peer is ingress-controller (a peer created while ingress analysis)
+const ingressPodName = "ingress-controller"
+
+func isIngressControllerPeer(peer eval.Peer) bool {
+	return strings.Contains(peer.String(), ingressPodName) // peer.String() == "{"+ingressPodName+"}"
+}
+
```

Comment:
why not comparing to a more specific string `{ingress-controller}`?

Reply:
changed